### PR TITLE
adds a confirmation to malf AI shunting

### PR DESCRIPTION
**Pull request by TheRyeGuyWhoWillNowDie**
## About The Pull Request

lol
## Why It's Good For The Game

people's rounds shouldn't be ruined from misclicking the button that's right next to the "progress as an antagonist" button

## Changelog
:cl:
qol: adds a confirmation to malf AI shunting into APCs
/:cl:

### CODE REVIEW

**Comment by Loafin34** [comment]
ishunt pr

File: code/modules/power/apc/apc_malf.dm [hunk 1]
```diff
@@ -36,6 +36,9 @@
 		return
 	if(!is_station_level(z))
 		return
+	var/confirm = tgui_alert(malf, "Are you sure that you want to shunt? This will take you out of your core!", "Shunt to [name]?", list("Yes", "No"))
+	if(confirm != "Yes")
+		return
```
Comment (san7890) [inline code/modules/power/apc/apc_malf.dm:41]: malfoccupy isn't allowed to sleep. The way to fix this is to put this tgui_alert and everything below that into a new proc, then use `INVOKE_ASYNC()` to call that proc. code should have a few examples.

**Review by san7890 — CHANGES_REQUESTED** [review]

**Comment by Admiral-Gigaton** [comment]
Thank fuck, this ruined my almost perfect achievement of doomsday, when there could be 2 ais naturally (fake AI and real AI)

File: code/modules/power/apc/apc_malf.dm (hunk 1 above)
Comment (TheRyeGuyWhoWillNowDie) [reply code/modules/power/apc/apc_malf.dm:41]: makes sense

**Comment by TheRyeGuyWhoWillNowDie** [comment]
ok we're back on

**Pull request merged**